lambda/app.py

```diff
@@ -342,11 +189,10 @@ def lambda_handler(event, context):
     
     catalog = GlueCatalog(glue_db_name)
     table = catalog.load_table((glue_db_name, glue_table_name))
-    logger.info(f"current snapshot id={table.metadata.current_snapshot_id}")
     snapshot = table.metadata.snapshot_by_id(table.metadata.current_snapshot_id)
```

Comment:
you can use table.current_snapshot() this will bring the current snapshot instance, why do you need the id if you can just get the current one??

Reply:
Correct, I'm using the suggested version in other places. Will update 👍